content/docs/join_us/recruitment_FAQs.md

```diff
@@ -198,44 +403,23 @@ The best thing you could do to be prepared is to eat well, take some long walks
-
-- Formal support for professional development via learning and development plans each team member develops with their line manager during the probation process, annual objective setting and regular 1-2-1s.
-  These learning plans feed into the projects people are allocated to and the roles and responsibilities they undertake within these.
-
-- Support to team members with their development outside of projects, providing 20% time that is split evenly between self-directed development and getting involved in one of the group's or institute's service areas.
-  We support team members to attend conferences, workshops and courses, and there are many seminar series and interest groups at the Turing that team members are encouraged to get involved with, including the team’s own weekly [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}).
-  We also support team members making contributions to the wider ecosystem as part of their 20% time.
-  Examples include contributing to external [Open Source Hacksessions]({{< relref "docs/regular_events/open_source_hacksessions.md" >}}) and contributing to community groups like the [Society for Research Software Engineering]({{< relref "docs/onboarding/society_of_research_software_engineering.md" >}}).
-
-- The opportunity to get involved in non-project work through one of the team's service areas, some of which support the running of the team and some of which support the wider research community at the institute (for instance Recruitment, Planning and finance, Training, EDI, Communications, Research computing support, Research programme liaison).
-  Some service areas require significant support and will have core team members who have chosen these service areas instead of a full project workload, but all team members have a 10% allocation to volunteer for a service area of their choice.
-
-- The opportunity to contribute to how the group works and grows.
-  We run the team in an open, consultative and collaborative manner and almost all of our decision making on how we organise ourselves and develop the team is done in the open on our group GitHub with the opportunity for anyone in the team to contribute.
-
-- [Other benefits](https://www.turing.ac.uk/work-turing/why-work-turing/employee-benefits) include private healthcare and a 13% contribution to a defined contribution pension scheme (3% from the employee, 10% from the Turing).
+Each year, there is a cost of living increase which is applied to all Turing employees.
+On top of this, we also have annual pay reviews, where we actively review the performance and development of each team member against the expectations of their position within their seniority band, making salary adjustments where these expectations are being exceeded.
```

Comment:
Maybe add a comment about applying to more senior roles within the team?

"We also hold annual internal promotion rounds to allow people to move up to the next level of seniority when they are ready. There is no cap on the number of junior, standard and senior roles, so promotion is not limited by availability"